net/respondd-module-airtime/src/ifaces.c

```diff
@@ -0,0 +1,64 @@
+#include <sys/socket.h>
+#include <linux/nl80211.h>
+#include <netlink/netlink.h>
+#include <netlink/genl/genl.h>
+#include <netlink/genl/ctrl.h>
+#include <net/if.h>
+#include <stdlib.h>
+
+#include "ifaces.h"
+
+static int iface_dump_handler(struct nl_msg *msg, struct iface_list **arg) {
```

Comment:
Please change the second parameter type to `void *` and remove the *nl_recvmsg_msg_cb_t* cast below. ISO C doesn't generally guarantee that all pointers have the same size, alignment or representation (only that any pointer can be converted to `void *` and back), so doing a function pointer cast like the one below might not be entirely correct.

(Note that this is only a cosmetic issue in practice; I think POSIX will guarantee that data pointers have the same representation. I'm not entirely sure though.)